cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,70 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
+describe("Student Registration page", () => {
+  const firstName = ["John", "Emma", "William", "Sophia", "Michael"][
+    Math.floor(Math.random() * 5)
+  ];
+  const lastNames = [
+    "Smith",
+    "Johnson",
+    "Williams",
+    "Jones",
+    "Brown",
+    "Davis",
+    "Miller",
+    "Wilson",
+    "Moore",
+    "Taylor",
+  ];
+  const randomLastName =
+    lastNames[Math.floor(Math.random() * lastNames.length)];
+  const email = `user${Math.floor(Math.random() * 10000)}@example.com`;
+  const phoneNumber =
+    "0" +
+    Math.floor(Math.random() * 1000000000)
+      .toString()
+      .padStart(9, "0");
+  const subjects = ["Math", "Science", "English", "History", "Art"];
+  const randomSubject = subjects[Math.floor(Math.random() * subjects.length)];
+
   before(() => {
-    
+    cy.visit("/");
   });
 
-  it('', () => {
-    
+  it("Registration", () => {
+    cy.get('[placeholder="First Name"]').type(firstName);
+    cy.get('[placeholder="Last Name"]').type(randomLastName);
+    cy.get('[placeholder="name@example.com"]').type(email);
+    const genderList = cy.get(
+      "#genterWrapper > .col-md-9 > :nth-child(1), #genterWrapper > .col-md-9 > :nth-child(2), #genterWrapper > .col-md-9 > :nth-child(3)"
+    );
+    const randomIndex = Math.floor(Math.random() * 3);
+    genderList.eq(randomIndex).click();
```

Comment:
it's hard to read your test because of the mix of the test and variables. please move your test data to a separate file